api/app/signals/apps/questionnaires/rest_framework/serializers/public/location.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Vereniging van Nederlandse Gemeenten
+from rest_framework.serializers import ModelSerializer, SerializerMethodField
+
+from signals.apps.signals.models import Location
+
+
+class LocationModelSerializer(ModelSerializer):
+    stadsdeel = SerializerMethodField()
+
+    class Meta:
+        model = Location
+        geo_field = 'geometrie'
+        fields = (
+            'geometrie',
```

Comment:
We agreed to leave this as `geometrie` for compatibility reasons. Renaming means we have to rewrite the mapping components in the frontend that use `geometrie` (because that Dutch naming is present in other parts of the Signalen API).